content/300-guides/050-database/400-prototyping-schema-db-push.mdx

```diff
@@ -102,13 +102,13 @@ The following scenario demonstrates how to use `db push` to synchronize a new sc
     /* ... */
     ```
 
-5. Push the changes:
+5.  Push the changes:
```

Comment:
Nitpick: Why two spaces here?

Reply:
(Across the whole file here really)

Reply:
Prettier again....

Reply:
That would be an incorrect configuration then though for Markdown.